parsec/core/fs/workspacefs/file_transactions.py

```diff
@@ -111,7 +111,7 @@ async def _read_chunk(self, chunk: Chunk) -> bytes:
         return data[chunk.start - chunk.raw_offset : chunk.stop - chunk.raw_offset]
 
     async def _write_chunk(self, chunk: Chunk, content: bytes, offset: int = 0) -> int:
-        data = padded_data(content, offset, offset + chunk.stop - chunk.start)
+        data: bytes = padded_data(content, offset, offset + chunk.stop - chunk.start)
```

Comment:
I don't think that's necessary, probably an artefact

Reply:
forget to remove that one